src/main/kotlin/org/jetbrains/research/testspark/tools/llm/generation/PromptManager.kt

```diff
@@ -83,7 +83,7 @@ class PromptManager(
                     classesToTest = classesToTest.map(this::createClassRepresentation).toList(),
                     polymorphismRelations = polymorphismRelations,
                     promptConfiguration = PromptConfiguration(
-                        desiredLanguage = psiHelper.language.languageName,
+                        desiredLanguage = psiHelper.language.languageId,
```

Comment:
Maybe it's better to use just `.name` and `.id`? It's more common style for our project, examples: `TestCase` or `LLMPlatform`.